### DISCUSSION TITLE
Cannot run executable created on Apple Silicon on Apple Intel

### DISCUSSION THREAD

**Discussion by koegl** [pyinstaller/pyinstaller#6666] [Help] [answered]
I'm not sure if that's a bug or a 'feature,' but I created an executable on my Apple Silicon which runs just fine, but it doesn't run on Apple Intel (it says: Bad CPU type in executable)

Is this supposed to happen? Or can I specify somehow for pyinstaller to create an executable that can be run on Intel Macs?

**Comment by rokm** [comment, marked as answer]
> Is this supposed to happen?

Yes.

See #6650.
